crates/tap-agent/Cargo.toml

```diff
@@ -49,14 +49,14 @@ ractor = { version = "0.14", features = [
 tap_aggregator.workspace = true
 futures = { version = "0.3.30", default-features = false }
 bon = "3.3"
-test-assets = { path = "../test-assets" , optional=true}
+test-assets = { path = "../test-assets", optional = true }
 
 [dev-dependencies]
+my-crate = { package = "indexer-tap-agent", path = ".", features = ["test"] }
```

Comment:
Could we leave a comment on this for people coming to it for the first time, please?